fabtests/scripts/runmultinode.sh

```diff
@@ -31,6 +31,16 @@ while true; do
 			cleanup=true; shift ;;
 		-x|--xfer-method)
 			xfer_method="$2"; shift 2 ;;
+		-E|--env)
+			delimiter="="
+			value=${2#*$delimiter}
+			var=${2:0:$(( ${#2} - ${#value} - ${#delimiter} ))}EXPORT_STRING="export $var=\"$value\""
+			if [[ -z $EXPORT_ENV ]] ; then
+				EXPORT_ENV="$EXPORT_STRING ;"
+			else
+				EXPORT_ENV="$EXPORT_ENV $EXPORT_STRING ;"
+			fi
```

Comment:
No need to use conditional here. Just use the statement in the `else` branch directly.